tsconfig.json

```diff
@@ -0,0 +1,26 @@
+{
+  "compilerOptions": {
+    /* Visit https://aka.ms/tsconfig.json to read more about this file */
+
+    /* Basic Options */
+    "target": "ES2020" /* ES2020 should be safe to use in browsers */,
+    "module": "NodeNext" /* Required for ESM mode */,
+    "lib": ["DOM", "ESNext"] /* Specify library files to be included in the compilation. */,
+
+    /* Strict Type-Checking Options */
+    "strict": true /* Enable all strict type-checking options. */,
+
+    /* Module Resolution Options */
+    "moduleResolution": "NodeNext" /* Required for ESM mode, forces the .js extension for local imports */,
+    "allowSyntheticDefaultImports": true /* Allow default imports from modules with no default export. This does not affect code emit, just typechecking. */,
+    "esModuleInterop": true /* Enables emit interoperability between CommonJS and ES Modules via creation of namespace objects for all imports. Implies 'allowSyntheticDefaultImports'. */,
+
+    /* Advanced Options */
+    "skipLibCheck": true /* Skip type checking of declaration files. */,
```

Comment:
I think we should only enable this option if performance is an issue.

Reply:
Disabled it with a comment